src/public/index.html

```diff
@@ -14,5 +14,6 @@
     <div nav-bar></div>
     <div ui-view class="site"></div>
     <script src="https://cdn.jsdelivr.net/g/jquery@3.1.1,bootstrap@3.3.7"></script>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```

Comment:
CSS should be loaded in the <head> of the document. Might even be good to group it in with the normal JSDelivr of whatever stylesheets we loading from them.